mlir/lib/Dialect/Utils/ReshapeOpsUtils.cpp

```diff
@@ -8,6 +8,7 @@
 
 #include "mlir/Dialect/Utils/ReshapeOpsUtils.h"
 
+#include "mlir/Dialect/Arith/IR/Arith.h"
```

Comment:
Dialect/Utils must not depend on a specific dialect.

Reply:
Makes sense. The only reasonable place to put this would be in arith dialect utils.
Kind of a strange place for reshape utils, but the way it works out, it just needs to go there

Reply:
Moved to Arith/Utils. Thanks :)